src/upgrades.cairo

```diff
@@ -1,2 +1,4 @@
 mod upgradeable;
 mod interface;
+
+use upgradeable::Upgradeable;
```

Comment:
reopening 😅

Reply:
reopening 😅

Reply:
My bad, I quickly read this and thought you were talking about the doc page example. 

We are currently not exposing interfaces like this. We are exposing them as part of the `interface` module in every contract. Reexposing it here would break consistency unless we update it for every module.